splunk-otel-android/src/main/java/com/splunk/rum/RumInitializer.java

```diff
@@ -392,18 +404,37 @@ SpanExporter getToDiskExporter(SpanStorage spanStorage) {
     }
 
     // visible for testing
-    SpanExporter getCoreSpanExporter(String endpoint) {
+    SpanExporter getCoreSpanExporter() {
+        Supplier<SpanExporter> exporterSupplier = supplyZipkinExporter();
+        if (builder.shouldUseOtlpExporter()) {
+            exporterSupplier = supplyOtlpExporter();
+        }
         // return a lazy init exporter so the main thread doesn't block on the setup.
-        return new LazyInitSpanExporter(
-                () -> {
-                    return ZipkinSpanExporter.builder()
-                            .setEncoder(new CustomZipkinEncoder())
-                            .setEndpoint(endpoint)
-                            // remove the local IP address
-                            .setLocalIpAddressSupplier(() -> null)
-                            .setSender(buildCustomizedZipkinSender())
-                            .build();
-                });
+        return new LazyInitSpanExporter(exporterSupplier);
+    }
+
+    @NonNull
+    private Supplier<SpanExporter> supplyOtlpExporter() {
+        // TODO: Do we want/need to append the auth query parameter like zipkin?
```

Comment:
Presumably this will get nailed down as we collaborate with the backend team?

Reply:
Actually I got confirmation today that it works the same way, so I should update this. How about I do it in a follow-up PR?